osu.Game/Online/Leaderboards/LeaderBoardScoreV2.cs

```diff
@@ -0,0 +1,550 @@
+                            drawables[i].FadeIn(100 + i * 50);
+                    }
+                }
+            }
+        }
+
+        protected override bool OnHover(HoverEvent e)
+        {
+            updateState();
+            return base.OnHover(e);
+        }
+
+        protected override void OnHoverLost(HoverLostEvent e)
+        {
+            updateState();
+            base.OnHoverLost(e);
+        }
+
+        private void updateState()
+        {
+            foreground.FadeColour(IsHovered ? foregroundColour.Lighten(0.2f) : foregroundColour, transition_duration, Easing.OutQuint);
+            background.FadeColour(IsHovered ? backgroundColour.Lighten(0.2f) : backgroundColour, transition_duration, Easing.OutQuint);
+        }
+
+        #region Subclasses
+
+        private partial class DateLabel : DrawableDate
+        {
+            public DateLabel(DateTimeOffset date)
+                : base(date)
+            {
+                Font = OsuFont.GetFont(size: 16, weight: FontWeight.Medium, italics: true);
+            }
+
+            protected override string Format() => Date.ToShortRelativeTime(TimeSpan.FromSeconds(30));
+        }
+
+        private partial class ScoreComponentLabel : Container
+        {
+            private readonly (LocalisableString Name, LocalisableString Value) statisticInfo;
+            private readonly ScoreInfo score;
+
+            private FillFlowContainer content = null!;
+            public override bool Contains(Vector2 screenSpacePos) => content.Contains(screenSpacePos);
+
+            public ScoreComponentLabel((LocalisableString Name, LocalisableString Value) statisticInfo, ScoreInfo score)
+            {
+                this.statisticInfo = statisticInfo;
+                this.score = score;
+            }
+
+            [BackgroundDependencyLoader]
+            private void load(OsuColour colours, OverlayColourProvider colourProvider)
+            {
+                AutoSizeAxes = Axes.Y;
+                OsuSpriteText value;
+                Child = content = new FillFlowContainer
+                {
+                    AutoSizeAxes = Axes.Both,
+                    Direction = FillDirection.Vertical,
+                    Children = new Drawable[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Colour = colourProvider.Content2,
+                            Text = statisticInfo.Name,
+                            Font = OsuFont.GetFont(size: 12, weight: FontWeight.Bold),
+                        },
+                        value = new OsuSpriteText
+                        {
+                            Text = statisticInfo.Value,
+                            Font = OsuFont.GetFont(size: 19, weight: FontWeight.Medium),
+                        }
+                    }
+                };
+
+                if (statisticInfo.Name == EditorSetupStrings.ComboColourPrefix.ToUpper())
```

Comment:
Shouldn't use this string as it's a different context. Either way it should be "max combo" to conform with other places (i.e. just ignore what's on the design here).

Reply:
gotcha